content/en/platform/corda/5.0/application-networks/creating/mgm/register.md

```diff
@@ -152,4 +153,20 @@ Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -
 ```
 {{% /tab %}}
 {{< /tabs >}}
-If successful, you should see the `APPROVED` registration status.
+
+The returned status of the registration request will be one of the following options:
+
+* `APPROVED`: The MGM was onboarded successfully.
+* `NEW`: The member’s Corda cluster has accepted and persisted the registration request submitted via the REST API but is yet to process it.
+* `INVALID`: The member’s Corda cluster processed the registration request submitted via the API and determined the
```

Comment:
There's a mix between the usage of `the REST API` and `the API`. Should all references to the rest api use the same terminology (apologies if I introduced this inconsistency in the samples I provided)

Reply:
Updates to REST API